src/components/CollectionsPage/CollectionRow/CollectionRow.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Table, Icon } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { locations } from 'routing/locations'
+import { getCollectionType } from 'modules/collection/utils'
+import CollectionStatus from 'components/CollectionStatus'
+import CollectionImage from 'components/CollectionImage'
+import { Props } from './CollectionRow.types'
+import styles from './CollectionRow.module.css'
+
+export default class CollectionRow extends React.PureComponent<Props> {
+  handleTableRowClick = () => {
+    const { onNavigate, collection } = this.props
+    onNavigate(locations.collectionDetail(collection.id, getCollectionType(collection)))
+  }
+
+  render() {
+    const { collection, items } = this.props
+    const type = getCollectionType(collection)
+
+    return (
+      <Table.Row className={styles.CollectionRow} onClick={this.handleTableRowClick}>
+        <Table.Cell width={4}>
+          <div className={styles.imageColumn}>
+            <CollectionImage className={styles.image} collectionId={collection.id} />
```

Comment:
It could happen, although pretty rare (if uploaded using the client), that a thumbnail doesn't have the required height and width and this will make the image look cropped.
![Screen Shot 2022-03-31 at 11 47 18](https://user-images.githubusercontent.com/1120791/161084484-8a188cfd-9334-4841-a408-0e0204cd2c12.png)
This can be fixed by removing a couple of properties in the classes applied to the image by the `ImageWrapper`
```
.ItemImage.image-wrapper .image
```